components/places/src/import/fennec/bookmarks.rs

```diff
@@ -302,7 +301,16 @@ lazy_static::lazy_static! {
             b.keyword,
             b.tags,
             sanitize_timestamp(b.created),
-            sanitize_timestamp(b.modified),
+            -- items with tags may have the tags destroyed by sync, so we
```

Comment:
this sql probably allows modified to be earlier than created, which the api forbids

Reply:
Can we set it to `min(b.created, b.modified)` for items with tags? Or are you worried our creation time might still be newer than the server's last-modified time, and we'll end up losing tags for those?